_rules/image-has-name-23a2a8.md

```diff
@@ -24,7 +24,7 @@ The rule applies to HTML `img` elements or any HTML element with the [semantic r
 
 ## Expectation
 
-Each target element has an [accessible name](#accessible-name) that is not only [whitespace](#whitespace), or is marked as [decorative](#decorative).
+Each target element has an [accessible name](#accessible-name) that consists of characters other than [whitespace](#whitespace).
```

Comment:
For that rule, I suppress the "is marked as decorative" in the expectation because I don't understand why the "marked as decorative" is not part of the applicability? 

> The rule applies to HTML `img` elements or any HTML element with the [semantic role](#semantic-role) of `img` that is [included in the accessibility tree](#included-in-the-accessibility-tree) and that is not marked as [decorative](#decorative)

Reply:
@audreymaniez Because marking an image as decorative is a technique to satisfy the SC. Decorative images are still non-text content, and so they're applicable to the SC 1.1.1. You're right that it doesn't make a big difference, but I think it fits the intent of the SC a little better.

Reply:
So I put it back. Thank you for the answer